templates/zerver/help/analytics.md

```diff
@@ -1,9 +1,15 @@
-# View organization statistics
+# Usage statistics
 
 Zulip provides detailed analytics to help you see how you and your team are
-using Zulip.
+using Zulip. The summary section of the usage statistics page includes your
+organization's overall usage information such as the total number of users,
+guests, messages, and storage in use.
 
-### View organization statistics
+You will also find interactive graphs that provide a detailed breakdown of
+your organization's **Active Users**, **Messages sent by recipient type**,
+**Messages sent over time**, **Messages read over time**, and **Messages sent by client**.
+
```

Comment:
Hmm, maybe something like this?

> You will also find interactive graphs that provide a detailed breakdown of
your organization's number of active users, messages sent, messages received, and how these statistics have changed over time.